tests/test_json.py

```diff
@@ -27,9 +28,57 @@ def test_serialization():
     assert isinstance(result["actions"], list)
 
 
-def test_action_from_json():
+def test_action_from_json_maritime_trade():
     data = ["RED", "MARITIME_TRADE", ["SHEEP", "SHEEP", "SHEEP", "SHEEP", "ORE"]]
     action = action_from_json(data)
     assert action.color == Color.RED
     assert action.action_type == ActionType.MARITIME_TRADE
     assert action.value == ("SHEEP", "SHEEP", "SHEEP", "SHEEP", "ORE")
+
+
+def test_action_from_json_play_year_of_plenty_two_resources():
+    data = ["RED", "PLAY_YEAR_OF_PLENTY", ["WOOD", "BRICK"]]
+    action = action_from_json(data)
+    assert action.color == Color.RED
+    assert action.action_type == ActionType.PLAY_YEAR_OF_PLENTY
+    assert action.value == ("WOOD", "BRICK")
```

Comment:
NIT: There are enums in enums.py that assign `WOOD = "WOOD"`. I'd be nice to use them as for performance I am thinking in the future it will help (at the cost of debuggability </3) to make them integers and I'm hoping to ease the pain of that future change.

Reply:
Sounds good. Note, if I try to use ActionType and Color enums, the tests fail. I'll update to use the color enums